tpu-provisioner/README.md

```diff
@@ -8,20 +8,22 @@ The provisioning process starts with an unschedulable "leader" pod (pod with Job
 JobSet. Once the TPU slice is created, the remaining pods for each Job will be created and follow their leader pod
 onto the same slice it is running on.
 
-<img src="./docs/provisioning.excalidraw.png" width="50%"></img>
-
-Node Pools are cleaned up when the JobSet whose pods triggered the node pool creation is either completed, failed, or deleted.
+Node Pools are cleaned up when the JobSet whose pods triggered the node pool creation is either **completed, failed, or deleted**.
 
 ## Setup
 
-### Create a GKE Cluster with workload identity enabled
+### Create a GKE Cluster with workload identity enabled and a static cluster version
 
-The TPU Provisioner requires workload identity to be enabled. 
+The TPU Provisioner requires workload identity to be enabled, and cannot be on a release channel (auto upgrades
```

Comment:
Can we change `workload identity` to `Workload Identity Federation for GKE`?

The seems to be the official name from reading the docs at https://cloud.google.com/kubernetes-engine/docs/concepts/workload-identity.

Reply:
Sure, updated.